hssqlppp/src/Database/HsSqlPpp/Pretty.lhs

```diff
@@ -255,12 +255,17 @@ Conversion routines - convert Sql asts into Docs
 >                          _ -> "") ++ "function")
 >     <+> name nm
 >     <+> parens (sepCsvMap paramDefn args)
->     <+> text "returns" <+> typeName retType <+> text "as" <+> text "$$"
->     $+$ functionBody body
->     $+$ text "$$" <+> text "language"
+>     <+> text "returns" <+> typeName retType <+> text "as"
+>     $+$ text "$$"
+>     <> functionBody body
+>     <> text "$$"
```

Comment:
how does this look compared to `$+$`?

Reply:
when it was $+$ it would add \n in places there wasn't before